code/game/objects/items/bodybag.dm

```diff
@@ -134,4 +137,26 @@
 				user << "The controls are now [src.locked ? "locked." : "unlocked."]"
 			else
 				user << "\red Access denied."
-			return
\ No newline at end of file
+			return
+		//Medborg Bodybag Containment Unit Proc
+		if(istype(W, /obj/item/weapon/cyborg/bodybag_containment_unit))
+			BCU_store(W, user)
+
+
+//Medborg Bodybag Containment Unit Proc
+/obj/structure/closet/body_bag/proc/BCU_store(W as obj, mob/user as mob)

```

Comment:
What even? Why is this a proc on bodybags? This should be part of the bodybag containment unit code.